src/planscape/plan/migrations/0012_scenario_creation_time_scenario_max_budget_and_more.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.1.3 on 2023-02-15 23:41
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('conditions', '0004_get_condition_pixels'),
+        ('plan', '0011_project_creation_time'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
double checking here - is there also a field for the creator's notes (in the mocks)?

Reply:
i haven't added that yet, let work on this in a follow up